ansible/roles/postldap/defaults/main.yml

```diff
@@ -378,7 +397,7 @@ postldap__postfix__dependent_lookup_tables:
                       )
                     )
                   )'
-    result_attribute: 'uid, mail, mailAlias'
+    result_attribute: 'uid, mailAddress, mailAlternateAddress'
```

Comment:
When you use authentication with the email address it is no more working since there is a mismatch between the login used (the email address) and the return `uid` and postfix refuse to send the message.

Therefore, `ldap_smtpd_sender_login_maps` lookup table should return `mailAddress` and `mailAlternateAddress`.